test/jdk/com/sun/jdi/JdwpAttachTest.java

```diff
@@ -168,6 +173,15 @@ private static void setConnectorArg(Map<String, Connector.Argument> args, String
         arg.setValue(value);
     }
 
+    // Attach to localhost tries to connect to both IPv4 and IPv6 loopback addresses.
+    // But sometimes it causes interference with other processes which can listen on the same port but different
+    // loopback address.
+    // The method checks if the address is safe to test with current network config.
+    private static boolean addressIfSafeToConnectToLocalhost(InetAddress addr) {
```

Comment:
Do you really mean to name it "addressIfSafe..." rather than "addressIsSafe..."?

Reply:
Correct. Fixed.